From: Takashi Iwai <tiwai@suse.de>
Subject: Fix kABI breakage with ath9k MSI support
Patch-mainline: Never, kABI fix
References: FATE#326580

ath9k MSI support added some new fields to be protected for ksyms.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/net/wireless/ath/ath9k/hw.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/net/wireless/ath/ath9k/hw.h
+++ b/drivers/net/wireless/ath/ath9k/hw.h
@@ -977,9 +977,11 @@ struct ath_hw {
 	bool tpc_enabled;
 	u8 tx_power[Ar5416RateSize];
 	u8 tx_power_stbc[Ar5416RateSize];
+#ifndef __GENKSYMS__
 	bool msi_enabled;
 	u32 msi_mask;
 	u32 msi_reg;
+#endif
 };
 
 struct ath_bus_ops {
